frontend/src/containers/AdmissionsContainer/InnerTable.jsx

```diff
@@ -0,0 +1,59 @@
+import React, { useMemo } from "react";
+import { useTable, useSortBy, useBlockLayout } from "react-table";
+import { innerCols } from "./Columns";
+
+const AdmissionsInnerTable = ({ groupApplications }) => {
+  const columns = useMemo(() => innerCols, []);
+  const data = useMemo(
+    () =>
+      groupApplications.map((application) => ({
+        group: application.group,
+        text: application.text,
+      })),
```

Comment:
What is the function of this mapping? Wouldn't `() => groupApplications` work?

Reply:
Here I meant
```diff
const data = useMemo(
    () =>
+      groupApplications,
-      groupApplications.map(() => ({
-        group: groupApplications.group,
-        text: groupApplications.text,
-      })),
    [groupApplications]
  );
```
because the new variable contains exactly the same props as the old one, so I don't see the need to remove the other properties. Or is it beneficial to do that?

Reply:
ahh i see, my bad, yeah that works, changed it now